### PR TITLE
Update manifest

### PR DESCRIPTION
**Pull request by Thatoo**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -6,8 +6,9 @@
         "en": "Duniter client to manage wallets, certifications in a libre money.",
         "fr": "Client Duniter pour gérer ses portes-monnaie, les certifications au sein d’une monnaie libre."
     },
+    "version": "1.3.11~ynh1",
     "url": "https://github.com/duniter/cesium",
-    "license": "free",
+    "license": "GNU AGPL-3.0",
```
Comment (M5oul) [inline manifest.json]: You should strictly follow the [rule from the documentation about this field](https://yunohost.org/#/packaging_apps_manifest_en).

Comment (Thatoo) [reply manifest.json]: Ok so I should remove GNU to respect the value from the Identifier column from https://spdx.org/licenses/ .

**Comment by Thatoo** [comment]
Done!

File: manifest.json (hunk 1 above)
Comment (M5oul) [reply manifest.json]: You are using a deprecated licence string that can be found at the bottom of the website.

So, you have the choice between `AGPL-3.0-only` and `AGPL-3.0-or-later`.
I can tell you that this is `AGPLv3+` as Cesium license contains `later` statements.

Comment (Thatoo) [reply manifest.json]: Sorry. I changed to `AGPL-3.0-or-later` then

**Pull request merged**